WalletWasabi.Gui/Tabs/SettingsView.xaml

```diff
@@ -10,7 +10,7 @@
   </UserControl.Resources>
   <DockPanel LastChildFill="true">
     <StackPanel DockPanel.Dock="Top" IsVisible="{Binding IsModified}" Orientation="Horizontal" HorizontalAlignment="Center" Margin="0 5 10 5">
-      <TextBlock Text="Heads Up!" FontWeight="Bold" Classes="warningMessage" Margin="0 0 3 0" />
+      <TextBlock Text="Warning!" Foreground="IndianRed" FontWeight="Bold" Classes="warningMessage" Margin="0 0 3 0" />
```

Comment:
We are trying to move away from hard coded colours in the UI, please either declare a resource like: WarningColor, thats what Classes=warningMessage was doing.